Heat_Eqn_Parallel/CMakeLists.txt

```
@@ -0,0 +1,39 @@
+##
+#  CMake script for the MPI_heat_eqn tutorial program:
+##
+
+# Set the name of the project and target:
+SET(TARGET "MPI_heat_eqn")
+
+# Declare all source files the target consists of. Here, this is only
+# the one step-X.cc file, but as you expand your project you may wish
+# to add other source files as well. If your project becomes much larger,
+# you may want to either replace the following statement by something like
+#    FILE(GLOB_RECURSE TARGET_SRC  "source/*.cc")
+#    FILE(GLOB_RECURSE TARGET_INC  "include/*.h")
+#    SET(TARGET_SRC ${TARGET_SRC}  ${TARGET_INC})
+# or switch altogether to the large project CMakeLists.txt file discussed
+# in the "CMake in user projects" page accessible from the "User info"
+# page of the documentation.
+SET(TARGET_SRC
+  ${TARGET}.cc
+  )
+
+# Usually, you will not need to modify anything beyond this point...
+
+CMAKE_MINIMUM_REQUIRED(VERSION 3.3.0)
+
+FIND_PACKAGE(deal.II 9.4.0
```

```suggestion
CMAKE_MINIMUM_REQUIRED(VERSION 3.13.4)

FIND_PACKAGE(deal.II 9.5.0
```

See https://github.com/dealii/code-gallery/pull/216.

----

I did not have time to look at the program yet. Just a comment about your CMakeLists.txt.

---

This is still relevant.